packages/dom-adapters/src/BlockToolAdapter/index.ts

```diff
@@ -73,18 +130,31 @@ export class BlockToolAdapter {
    * @param input - input element
    * @param key - data key input is attached to
    */
-  #handleBeforeInputEvent(event: InputEvent, input: HTMLElement, key: DataKey): void {
+  #handleBeforeInputEvent = (event: InputEvent, input: HTMLElement, key: DataKey): void => {
     /**
      * We prevent all events to handle them manually via model update
      */
     event.preventDefault();
 
+    const nativeInput = isNativeInput(input);
```

Comment:
Better to call such variables with `is`, as `isNativeInput` is already declared, you could name it smth like
```suggestion
    const isInputNative = isNativeInput(input);
```